src/components/table.js

```diff
@@ -259,37 +280,52 @@ FlexTable.propTypes = {
   headerHeight: PropTypes.number,
   rowHeight: PropTypes.number,
   styleHeader: PropTypes.func,
-  styleCell: PropTypes.func
+  styleCell: PropTypes.func,
+  tableName: PropTypes.string,
+  sort: PropTypes.shape({
+    field: PropTypes.string,
+    direction: PropTypes.string
+  })
 }
 
 /**
  * A basic table with a header and flexible column widths. Intended for small amounts of data,
  * since it does not provide scrolling. See FlexTable for prop types.
  */
-export const SimpleFlexTable = ({ columns, rowCount, noContentMessage, hoverHighlight }) => {
-  return h(Fragment, [
-    div({ style: { height: 48, display: 'flex' } }, [
-      _.map(([i, { size, headerRenderer }]) => {
+export const SimpleFlexTable = ({ columns, rowCount, noContentMessage, hoverHighlight, tableName = 'data table', sort = null }) => {
```

Comment:
Hmm, that's a great point. I was trying to keep these changes as unobtrusive to developers as possible, and trying to avoid having to make changes to every table.

But on the other hand, I already changed all the other tables to add support for `aria-sort`, so that's probably not a big deal at this point.

And it would certainly be better to force devs to explicitly think about accessibility where they need to. Now that I think about it, this caption is sort of like adding `alt` text to an image: sure you could just use a default but that really defeats the purpose.

Ok, I'll make it a required field, and update all of the tables accordingly.